core/src/main/java/hudson/cli/handlers/JobOptionHandler.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2004-2009, Sun Microsystems, Inc.
```

Comment:
as this is a new file the date should be the current year - and the copyright owner should be who owns the copyright of this code (either you or the company you work for depending on when you did this code change and your employment contract)

e..g 
`Copyright (c) 2016, David McFarland` or `Copyright (c) 2016, Acme Inc.` (if you work for Acme Inc,)